entities/table_row.rb

```diff
@@ -0,0 +1,13 @@
+class TableRow
+  attr_accessor :name, :difficulty, :att_total, :att_used, :hints_total, :hints_used, :date
+
+  def initialize(name:, difficulty:, att_total:, att_used:, hints_total:, hints_used:, date:)
```

Comment:
Could you please refactore this class? you just pass `option` hash to `initialize`